packages/suite-data/files/translations/en.json

```diff
@@ -722,7 +722,7 @@
 	"TR_GAS_PRICE": "Gas price",
 	"TR_GAS_USED": "Gas used",
 	"TR_GATHERING_INFO": "Gathering information, please wait...",
-	"TR_GENERAL": "General",
+	"TR_GENERAL": "Application",
```

Comment:
This is suspicious.

Reply:
OTOH, this key is used on a single place in the settings menu. So this is most probably fine.,